pkg/sfu/connectionquality/connectionstats.go

```diff
@@ -1,25 +1,28 @@
 package connectionquality
 
-import (
-	"github.com/livekit/protocol/livekit"
-)
+import "sync"
 
 type ConnectionStat struct {
 	PacketsLost  uint32
 	Delay        uint32
 	Jitter       uint32
 	TotalPackets uint32
 	LastSeqNum   uint32
+	TotalBytes   uint64
+	NackCount    int32
+	PliCount     int32
+	FirCount     int32
+	Score        float64
```

Comment:
Sorry for increasing work here. Maybe a good idea to split this into two structs one which is needed for delta stats and one which is not. Reason being `Prev` is not fully filled out when it is set up. Thought it could make it clear what is required for delta. Later, if we need to add more to delta, it can be added to the correct struct and will make it easier to reason.